pkg/cmd/util/docker/docker.go

```diff
@@ -36,6 +38,34 @@ func (_ *Helper) GetClient() (client *docker.Client, endpoint string, err error)
 	return
 }
 
+// KubeDocker provides a wrapper to Kubernetes Docker interface
+// This wrapper is compatible with OpenShift Docker interface.
+type KubeDocker struct {
+	dockertools.DockerInterface
+}
+
+// Ping implements the DockerInterface Ping method.
+func (c *KubeDocker) Ping() error {
+	client, err := docker.NewClientFromEnv()
+	if err != nil {
+		return err
+	}
+	return client.Ping()
+}
+
+// GetKubeClient returns the Kubernetes Docker client.
+func (_ *Helper) GetKubeClient() (*KubeDocker, string, error) {
+	var endpoint string
+	if len(os.Getenv("DOCKER_HOST")) > 0 {
```

Comment:
@csrwng this is mostly affecting the new-app code as far I can tell, can you have a look?